### PR TITLE
Add imagePullSecrets to deployments

### PR DESCRIPTION
**Pull request by mirekys**
This PR allows to deploy revad & wopiserver with custom images pulled from private image registries.

https://kubernetes.io/docs/concepts/containers/images/#specifying-imagepullsecrets-on-a-pod

Useful for e.g. deploying customized builds of revad for debugging issues

### Contributing a Chart / update to an existing Chart

- [x] Run `helm lint` on the chart dir.
- [x] (Update) Bump the `Chart.yaml` version before merging, to release it as a new version.
- [x] (Update) If the PR includes new configurable parameters in the chart's `values.yaml`. Add documentation in the appropiate README.

### CODE REVIEW

**Comment by david-antos** [comment]
Could you kindly give this PR a priority? It is currently blocking debugging of OC/NC applications (we need to set up the devel branch to continue). Thanks!

File: revad/Chart.yaml [hunk 1]
```diff
@@ -23,6 +23,7 @@ keywords:
   - sync-and-share
 annotations:
   artifacthub.io/changes: |
+    - "Add imagePullSecrets to deployment"
     - "Update revad image to v1.18.0"
```
Comment (SamuAlfageme) [inline revad/Chart.yaml:27]:
```suggestion
```

You can remove this line from the changelog as it was already released.

File: wopiserver/Chart.yaml [hunk 2]
```diff
@@ -18,8 +18,9 @@ keywords:
   - efss
 annotations:
   artifacthub.io/changes: |
+    - "Add imagePullSecrets to deployment"
     - "update image to cs3org/wopiserver:v8.3.3"
```
Comment (SamuAlfageme) [inline wopiserver/Chart.yaml:22]:
```suggestion
```

Same as above

File: wopiserver/values.yaml [hunk 3]
```diff
@@ -4,7 +4,7 @@ image:
   repository: cs3org/wopiserver
   tag: v8.3.3
   pullPolicy: IfNotPresent
-
+  pullSecrets: []
```
Comment (SamuAlfageme) [inline wopiserver/values.yaml:7]:
```suggestion
  pullSecrets: 
    []
```

to harmonize with the rest of the changes in this PR?

File: revad/values.yaml [hunk 4]
```diff
@@ -4,6 +4,7 @@ image:
   repository: cs3org/revad
   tag: v1.18.0
   pullPolicy: Always
+  pullSecrets: []
```
Comment (SamuAlfageme) [inline revad/values.yaml:7]:
```suggestion
  pullSecrets: 
    []
```

**Review by SamuAlfageme — APPROVED** [review]
@mirekys a few nitpicks on the format/changelog. Otherwise looks good and can be merged

File: revad/Chart.yaml (hunk 1 above)
Comment (mirekys) [reply revad/Chart.yaml]: fixed by 5e1a2568bf52ffaa97b46f1f91141f7616da6671

**Pull request merged**